iothub/device/src/Transport/Amqp/AmqpUnit.cs

```diff
@@ -542,21 +580,22 @@ public void Dispose()
 
         private void Dispose(bool disposing)
         {
-            if (_disposed) return;
+            if (_disposed)
```

Comment:
Nit: 2 lines instead of 8 makes it clearer:

```c#
if (disposed) return;
disposed = true;
```

Reply:
Code in next iteration